runtimes/alibi-explain/mlserver_alibi_explain/common.py

```diff
@@ -51,6 +57,19 @@ def remote_predict(
     return InferenceResponse.parse_raw(response_raw.text)
 
 
+def remote_metadata(url: str) -> MetadataModelResponse:
+    """Get metadata from v2 endpoint"""
+    response_raw = requests.get(url)
```

Comment:
Not that it matters much, but should we make this call asynchronous?

Reply:
thats a good point. having a look at the caller `load`, it has also `load_explainer` that we need  to do async so that the entire function (i.e `load`) is truly async. 

I am tempted however to leave it as a separate ticket?